repository/src/main/resources/alfresco/ibatis/org.alfresco.repo.domain.dialect.Dialect/node-common-SqlMap.xml

```diff
@@ -1147,6 +1147,21 @@
             parentNode.id = #{parentNode.id} and
             a.child_node_id IS NULL
     </select>
+
+    <select id="select_RootGroups" parameterType="ChildAssoc" resultType="java.lang.String">

```

Comment:
This query looks more generic than just user groups, would it be possible to use it for something else too? Maybe the name could be made more generic?

Reply:
Initially I thought that the name should be specific but now I realized that the queries can be used for other things. Changed the name to: **select_LocalNameListOfChildAssocsWithoutParent**